pinot-core/src/main/java/org/apache/pinot/core/io/reader/impl/ConstantSingleValueSortedIndex.java

```diff
@@ -59,26 +42,32 @@ public int getInt(int row) {
   }
 
   @Override
-  public int getInt(int rowId, SortedIndexReaderImpl.Context context) {
+  public int getInt(int rowId, ReaderContext context) {
     return 0;
   }
 
   @Override
-  public void close()
-      throws IOException {
+  public void readValues(int[] rows, int rowStartPos, int rowSize, int[] values, int valuesStartPos) {
+    Arrays.fill(values, valuesStartPos, valuesStartPos + rowSize, 0);
```

Comment:
Good catch. Could you fix another older misuse too:
https://github.com/apache/incubator-pinot/blob/master/pinot-core/src/main/java/org/apache/pinot/core/io/reader/impl/IntSingleValueDataFileReader.java#L54

Reply:
Removed it